# TODO: Find a better place for such errors
class UserNotFoundError(Exception):
    pass
